Makefile

```diff
@@ -6,23 +6,22 @@ GOLIST=go list
 GOFMT=gofmt
 BUILD_GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURPATH)/cmd
 
-IMAGE_BUILDER_OPTS=
-IMAGE_BUILDER?=imagebuilder
-IMAGE_BUILD=$(IMAGE_BUILDER)
-export IMAGE_TAG_CMD?=docker tag
+IMAGE_BUILD_OPTS=${MAGE_BUILDER_OPTS:- ""}
+DOCKERFILE=${DOCKERFILE:- "Dockerfile"}
+DOCKERFILE=Dockerfile
 
 export APP_NAME=sriov-network-operator
 APP_REPO=github.com/openshift/$(APP_NAME)
 TARGET=$(TARGET_DIR)/bin/$(APP_NAME)
-export IMAGE_TAG=quay.io/pliurh/$(APP_NAME):latest
+export IMAGE_TAG=quay.io/openshift/$(APP_NAME):latest
```

Comment:
The image in openshift repo is built by openshift CI system. Building image with Makefile shall only be use for test purpose. So I think we shall use different tag for the test image. Maybe we can use an environment variable here.